### PR TITLE
Add more tests for recommendations type

### LINKED ISSUE #182: Parse old style ISO Recommendation Addendums
```
ISO/R 91-1970 — Addendum 1
ISO/R 91:1970/Add 1:1975
ISO/R 91:1970/ADD 1:1975
ISO/R 194:1969/Add 4
ISO/R 298:1963/Add 1
ISO/R 947:1969/Add 1:1969
```

### LINKED ISSUE #183: Create new Identifier class for "ISO/R" Recommendations
## Basic pattern

```
ISO/R 1:1951
ISO/R 2:1951
ISO/R 3:1953
ISO/R 4:1953
ISO/R 5:1954
ISO/R 6:1954
ISO/R 19:1956
ISO/R 20:1956
ISO/R 38:1957
ISO/R 39:1957
ISO/R 40:1957
ISO/R 51:1957
```

## Parts (standard style)

Parts are listed in the pattern of:
* ` ISO/R {number}-{part number in roman numerals}:{year}`

```
ISO/R 93-3:1969
ISO/R 93-1:1963
ISO/R 93-2:1969
ISO/R 105-1:1959
ISO/R 105-6:1972
ISO/R 105-7:1975
ISO/R 105-1:1959
ISO/R 105-2:1963
ISO/R 105-3:1963
ISO/R 300-3:1968
ISO/R 355-8:1973
```

## Parts (old style)

Parts are listed in the pattern of:
* ` ISO/R {number}/{part number in roman numerals}-{year}`

We only need to support the parsing of these identifiers.

```
ISO/R 93/1-1963
ISO/R 93/II-1969
ISO/R 93/III-1969
ISO/R 105/I
ISO/R 105/II
ISO/R 105/III
ISO/R 300/III-1968
ISO/R 355-VI:1970
ISO/R 657/I
ISO/R 657/II
ISO/R 657/III
ISO/R 657/IV
ISO/R 710/III
```

## Addendums

See #182 .

### PR DESCRIPTION
**Pull request by mico**
closes #183 
closes #182

### CODE REVIEW

File: spec/pubid_iso/identifier_spec.rb [hunk 1]
```diff
@@ -647,6 +647,33 @@ module Pubid::Iso
...
+    context "ISO/R 300/III-1968" do
+      let(:original) { "ISO/R 300/III-1968" }
+      let(:pubid) { "ISO/R 300-III-1968" }
+
+      it_behaves_like "converts pubid to pubid"
+    end
+
+    context "ISO/R 657/IV" do
+      let(:original) { "ISO/R 657/IV" }
+      let(:pubid) { "ISO/R 657-IV" }
```
Comment (ronaldtse) [inline spec/pubid_iso/identifier_spec.rb:672]: The formal (new style) pubid is actually "[ISO/R 657-4:1969](https://www.iso.org/standard/4795.html)"

File: spec/pubid_iso/identifier_spec.rb [hunk 2]
```diff
@@ -647,6 +647,33 @@ module Pubid::Iso
...
+    context "ISO/R 93/1-1963" do
+      let(:original) { "ISO/R 93/1-1963" }
+      let(:pubid) { "ISO/R 93-1-1963" }
+
+      it_behaves_like "converts pubid to pubid"
+    end
+
+    context "ISO/R 300/III-1968" do
+      let(:original) { "ISO/R 300/III-1968" }
+      let(:pubid) { "ISO/R 300-III-1968" }
```
Comment (ronaldtse) [inline spec/pubid_iso/identifier_spec.rb:665]: The official pubid is [ISO/R 300-3:1968](https://www.iso.org/standard/4232.html)

File: spec/pubid_iso/identifier_spec.rb [hunk 3]
```diff
@@ -647,6 +647,33 @@ module Pubid::Iso
...
 
+    context "ISO/R 93-3:1969" do
+      let(:pubid) { "ISO/R 93-3:1969" }
+
+      it_behaves_like "converts pubid to pubid"
+    end
+
+    context "ISO/R 93/1-1963" do
+      let(:original) { "ISO/R 93/1-1963" }
+      let(:pubid) { "ISO/R 93-1-1963" }
```
Comment (ronaldtse) [inline spec/pubid_iso/identifier_spec.rb:658]: The official pubid is [ISO/R 93-1:1963](https://www.iso.org/standard/3752.html).

We will need to implement a Roman numeral conversion for the old style part numbers.

**Review by ronaldtse — CHANGES_REQUESTED** [review]

**Comment by ronaldtse** [comment]
@mico can you help implement the above comments? Thanks.

**Comment by mico** [comment]
> @mico can you help implement the above comments? Thanks.

@ronaldtse done

**Comment by ronaldtse** [comment]
Thank you @mico !

**Review by ronaldtse — APPROVED** [review]

**Pull request merged**